infra/cmake/packages/RuySourceConfig.cmake

```diff
@@ -9,11 +9,13 @@ function(_RuySource_import)
 
   # NOTE Downloads source from latest ruy library (2020-04-10)
   envoption(EXTERNAL_DOWNLOAD_SERVER "https://github.com")
-  set(RUY_URL ${EXTERNAL_DOWNLOAD_SERVER}/google/ruy/archive/2e2658f964638ab7aa562d4b48b76007d44e38f0.tar.gz)
-  ExternalSource_Download("ruy" ${RUY_URL})
+  envoption(RUY_URL ${EXTERNAL_DOWNLOAD_SERVER}/google/ruy/archive/2e2658f964638ab7aa562d4b48b76007d44e38f0.tar.gz)
+  ExternalSource_Download(RUY
+    DIRNAME RUY
+    URL ${RUY_URL})
 
-  set(RuySource_DIR ${ruy_SOURCE_DIR} PARENT_SCOPE)
-  set(RuySource_FOUND ${ruy_SOURCE_GET} PARENT_SCOPE)
```

Comment:
`ExternalSource_Download` don't set `${PREFIX}_SOURCE_GET`

Reply:
@hseok-oh Thanks! I didn't know that.